themes/edx.org/lms/templates/header.html

```diff
@@ -125,19 +125,21 @@ <h1 class="hd logo-header">
         <div class="right nav-courseware list-inline">
           <div class="item nav-courseware-01">
             % if not settings.FEATURES['DISABLE_LOGIN_BUTTON']:
-              % if course and settings.FEATURES.get('RESTRICT_ENROLL_BY_REG_METHOD') and course.enrollment_domain:
-                <a class="btn btn-login" href="${reverse('course-specific-login', args=[course.id.to_deprecated_string()])}${login_query()}">${_("Sign in")}</a>
-              % else:
-                <a class="btn btn-login" href="/login${login_query()}">${_("Sign in")}</a>
-              % endif
+                % if not combined_login_and_register:
```

Comment:
@douglashall Here I have a question with your `or` operator suggestion. If someone explicitly disable the login button e.g. `DISABLE_LOGIN_BUTTON = True` so in that case the user would be able to see the `Login / Register` links in header for other pages with the `or` condition.
Let me know would it be the desired behavior or keep it as it is above ? Above I am giving precedence to `DISABLE_LOGIN_BUTTON` flag.